### ISSUE TITLE
Documentation: Create more examples

### ISSUE THREAD

**Issue by ndabAP** [ndabAP/vue-command#179]
The [main site](https://ndabap.github.io/vue-command/) has only one big example. Create multiple examples how to use the library with code examples. A complete redesign is absolutely possible.

**Comment by shanerowden** [comment]
Just to add, because I've been looking over your code and seeing what I can maybe do with it for my needs, but still being kind of new-ish at Vue -- I think more helpful than more example commands would be, "To make a new command that is not writing a string into a component, you need to look here in this file in and take x into account, look in y file and take that into account. 

The Auto Complete stuff seems the most heavy I guess. It took me a while to figure out what providing and injecting was but I got it eventually. Either way, examples would be cool, but mainly only if they are instructional in how to add to the engine without breaking it.

**Comment by shanerowden** [comment]
Actually, many of the things you said in the README confused me thoroughly, being pretty new to JS frameworks. But after I looked through all your code and went back and read it, I understand.

**Comment by ndabAP** [comment]
>The Auto Complete stuff seems the most heavy I guess.

Definitely, and it's even not yet part of the core. You need to copy paste it from the examples. The plan was to fully integrate it but that needs careful planing. I didn't find the time so far.

>Actually, many of the things you said in the README confused me thoroughly, being pretty new to JS frameworks. But after I looked through all your code and went back and read it, I understand.

I'd be very happy for contributions because most stuff is only written by me which means it's only from my perspective.

What I meant with this was more examples at the hosted site here: https://ndabap.github.io/vue-command.

**Comment by wes-kay** [comment]
Apologies for Necro,

> The Auto Complete stuff seems the most heavy I guess. It took me a while to figure out what providing and injecting was but I got it eventually. Either way, examples would be cool, but mainly only if they are instructional in how to add to the engine without breaking it.

I Was just about to do auto complete (tab complete on commands), but thought to check here, are you saying there is functionality already for auto completing commands?
